package.json

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
master branch is old. This PR is supposed to be based on https://github.com/OAK-Foundation/xcm-demo/pull/28 which is created 4 days ago.

Reply:
Merge #28 first and then sync master.